iModelCore/ECPresentation/PublicAPI/ECPresentation/Content.h

```diff
@@ -1285,6 +1288,11 @@ struct EXPORT_VTABLE_ATTRIBUTE ContentDescriptor : RefCountedBase
     //! Should only distinct values be returned
     bool OnlyDistinctValues() const {return HasContentFlag(ContentFlags::DistinctValues);}
 #endif
+
+    //! Set if this descriptor is created using a different from the input ruleset
+    void SetDifferentFromInputRuleset(bool value) {m_differentFromInputRuleset = value;}
+    //! Is the descriptor created using a different from the input ruleset
+    bool IsDifferentFromInputRuleset() const {return m_differentFromInputRuleset;}

```

Comment:
maybe just set this to `true` in `SetRuleset`?

Reply:
Also, `IsDifferentFromInputRuleset` sounds like we're comparing descriptor with a ruleset. Maybe rename to `UsesModifiedRuleset`?

Reply:
fixed